cllc-public-app/ClientApp/src/app/components/sep/sep-application/sep-application.component.ts

```diff
@@ -24,24 +24,17 @@ export const SEP_APPLICATION_STEPS = ["applicant", "eligibility", "event", "liqu
 export class SepApplicationComponent implements OnInit {
   faCheck = faCheck;
   securityScreeningEnabled: boolean;
-  applicationId: string;
+  applicationId: number;
```

Comment:
This should be a string to be consistent with other fields that store a GUID.  dashes are valid in a guid for example, and that would break a true number.

Reply:
Agree with George here

Reply:
I will rename this field and comments. This is the id of the application in the browser indexeddb. The Id you are thinking of is the specialEventId in the SepApplication model.